CI.hs

```diff
@@ -172,6 +172,11 @@ buildDists ghcFlavor resolver = do
         pkg_ghclib_parser = "ghc-lib-parser-" ++ version
 
     -- Make and extract an sdist of ghc-lib-parser.
+    appendFile "ghc/hadrian/stack.yaml" $ unlines ["ghc-options:","  \"$everything\": -O0 -j"]
+    -- Temporary (09/14/2012) : Until
```

Comment:
I don’t think that’s a good idea. The MR is already approved and assigned to marge. I don’t see what we gain by patching this now only to have to undo the patch in a few days.

Reply:
As soon as it lands I'll undo it. For now though I can't uncover the other issues without it :)

Reply:
^ I promise!

Reply:
My point is why don’t we just wait until it’s patched upstream? We don’t even use the GHC master version in DAML so waiting a few days seems totally reasonable.

Reply:
Because it's my experience that keep it up-to-date incrementally is easier than letting all the breaking changes stack up! I mean, it's no big deal but keep it working helps with my anxiety.

Reply:
In this case, I could remove this patch and just commit the rest but of course, it would then show as broken in CI. I don't know if that's preferable?

Reply:
Fair enough but I’m not entirely convinced that we really need to keep the frequency we have right now.

Reply:
The frequency with which I sync with upstream? I tend to do so once a week. This one cost me ~3 or 4 hours. I've tried many strategies - this one is working best for me 🤷‍♀